src/main/resources/log4j.properties

```diff
@@ -0,0 +1,11 @@
+# Set root logger level to DEBUG and its only appender to A1.
```

Comment:
Why log4j instead of logback?

Reply:
habit, is there a compelling reason to use logback over log4j?

Reply:
Yeah, I think logback is considered to have deprecated log4j, just like SLF4J has deprecated commons-logging - e.g. https://stackoverflow.com/questions/178215/log4j-vs-logback . I think logback is considered as the more modern choice.